templates/full-time-program.html

```diff
@@ -224,11 +225,13 @@ <h1>Eligibility</h1>
         <li>have attended at least one of the study night groups for the application process
         </li>
       </ul>
+      <!-- Do we need to say anything about the Fall 2023 application process here? -->
       <p class="justify">If you don't get in the first time, you can apply two more times for a maximum of three times,
-        and they do not have to be consecutive application rounds. If you have any special circumstances that make you
-        question your eligibility for Techtonica’s
-        program, please feel free to contact us and we will be happy to discuss with you. If you are interested in
-        Techtonica but cannot participate in the next cohort, please fill out our
+        and they do not have to be consecutive application rounds. Fall 2023 applicants that passed their final round of 
+        the incomplete Fall 2023 application process may receive invitation to continue participating the July 2024 
```

Comment:
"may receive invitation" — "will be invited"
"participating the" —> "participating in the"

Reply:
### ✅ Addressed in [476c912](https://github.com/Techtonica/techtonica.org/pull/336/commits/476c912449d5aed0a1343b11235d47ed52ed07db).